run_tests_language

```diff
@@ -16,6 +17,7 @@ for t in ${tests}; do
 
   echo -n "...    ${language} $name"
 
+  sed -i "s/#!\/usr\/bin\/expect/#!${expect_path}\/expect/" "./tests/${language}/$t"
```

Comment:
```suggestion
  sed -i "s;#!/usr/bin/expect;#!${expect_path}/expect;" "./tests/${language}/$t"
```

does this work? that way we don't need the double-escaping.

Reply:
Let me try that!